### DISCUSSION TITLE
Did next-pwa update the service worker cache by default when prod is updated?

### DISCUSSION THREAD

**Discussion by tonyvugithub** [shadowwalker/next-pwa#203] [Q&A] [answered]
Hi @shadowwalker, my manager asked me a question that I could not seem to find in the README. Would the PWA updated itself if connected to the internet if there is a change in prod. I'm talking about big chance like a redesign of the website or something. Thank you very much for your information.

**Comment by shadowwalker** [comment, marked as answer]
Yes, it will update

**Reply by tonyvugithub** [reply]
Thank you very much!
